OADP-447: PVC Snapshot restore does not work if PVC requested size does not match disk capacity (#45986)
openshift/openshift-docs: modules/oadp-creating-restore-cr.adoc

* You must install the OpenShift API for Data Protection (OADP) Operator.
* The `DataProtectionApplication` CR must be in a `Ready` state.
* You must have a Velero `Backup` CR.

> * Ensure that PV capacity at backup time matches the requested size, adjusting the requested size to match capacity.

Reviewer: I would suggest removing the bit after the comma. It seems repetitive of what you have in the first part of the sentence. 
```suggestion
* Ensure that persistent volume (PV) capacity at backup time matches the requested size.
```

Author: I raised this issue with the SME who said that "It may not be obvious to average user who maybe conscious of space requested."

Reviewer: I think keeping the clause as is still presents confusion. I have played with the wording a bit. what do you think of:
```suggestion
* Ensure to adjust the requested size so the persistent volume (PV) capacity matches the requested size at backup time.
```

> * Ensure that PVC capacity at backup time matches the requested size, adjusting the requested size to match capacity.

Author: Fixed.

> * Ensure that PVC capacity at backup time matches the requested size, adjusting the requested size to match capacity.

Reviewer 2: lgtm
